client/src/components/Draggable/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useRef, useCallback } from 'react'
+
+const Draggable = (
+  {
+    children,
+    position,
+    updatePosition = null,
```

Comment:
It's probably more intuitive and correct to name the callback function `onPositionChange`